audits/code4rena-panoptic-next-core-summary.md

```
@@ -0,0 +1,92 @@
+# Panoptic: Next Core Findings & Analysis Report
+
+*Source: [code4rena.com/reports/2025-12-panoptic-next-core](https://code4rena.com/reports/2025-12-panoptic-next-core)*
+
+#### 2026-03-09
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)